medicines/web/src/services/search-results-loader.test.ts

```diff
@@ -0,0 +1,23 @@
+import { makeCursor } from './search-results-loader';
+
+describe(makeCursor, () => {
+  it.each([2, 4, 5, 10, 15, 20])('offsets first result by -1', pageSize => {
+    const cursor = makeCursor(1, pageSize);
+    expect(cursor).toBe('LTE=');
+  });
+
+  it.each([
+    [2, 10, 'OQ=='],
+    [3, 5, 'OQ=='],
+    [2, 2, 'MQ=='],
+    [3, 1, 'MQ=='],
+    [2, 3, 'Mg=='],
+    [4, 1, 'Mg=='],
```

Comment:
took my brain a little while to figure out but yes the ones that are the same are due to being a different page size - ie page 3 of size 5 is the 11th item onwards, as is page 2 of size 10